### DISCUSSION TITLE
Windows Subsystem for Android (WSA) Version 2307.40000.6.0 has been released

### DISCUSSION THREAD

**Discussion by Almighty188** [LSPosed/MagiskOnWSALocal#668] [General]
Windows Subsystem for Android (WSA) Version 2307.40000.6.0 has been released

1) On the Windows side, download it from:
https://github.com/MustardChef/WSAPackages/releases/download/WSA_2307.40000.6.0/MicrosoftCorporationII.WindowsSubsystemForAndroid_2307.40000.6.0_neutral_._8wekyb3d8bbwe.Msixbundle
with a big thanks to @MustardChef for providing the file archive.
2) Rename the downloaded file to wsa-WIF.zip
3) Delete everything inside the download folder for MagiskOnWSALocal
4) Place the wsa-WIF.zip file in the MagiskOnWSALocal download folder
*** Note *** Build is broken per [Bug] version 2306 build failed [Bug] version 2306 build failed #615 but can be built by manually making changes to scripts/build.sh with many thanks to @YT-Advanced as seen here:
https://github.com/LSPosed/MagiskOnWSALocal/pull/620/files
5) In Linux, Build MagiskOnWSALocal as usual and remember to choose "insider fast" for WSA release type.

**Comment by im1kh** [comment]
https://github.com/LSPosed/MagiskOnWSALocal/discussions/655#discussioncomment-6891321
